src/upload.ts

```diff
@@ -40,7 +40,7 @@ export async function upload (args: string[], internal = false) {
     if (!internal) {
       console.log(colors.green('uploaded:'), destination)
     } else {
-      console.log(colors.green(`${path.relative('.', filepath)}:`), destination)
+      console.log(colors.green(`${String(path.relative('.', filepath))}:`), destination)
```

Comment:
`path.relative` should be typed to return a `string`, so again like with `path.join` I don't get why this change was made, and if there's a way to get rid of it, it would be appreciated